tests/e2e/support/objects/app-files/link/actions.ts

```diff
@@ -44,6 +46,15 @@ export type deleteLinkArgs = {
   name: string
 }
 
+export type getPublicLinkVisibilityArgs = {
+  page: Page
+  linkName: string
+}
+
+export type getLinkEditButtonVisibilityArgs = {
+  page: Page
+  linkName: string
+}
```

Comment:
both types have the same keys inside. can we use one single type with a more generic name?